### DISCUSSION TITLE
Yarp 1.1.0 timeframe

### DISCUSSION THREAD

**Discussion by pinkfloydx33** [dotnet/yarp#1557] [General] [answered]
Is there any time frame for the release of the 1.1.0 version of `Yarp.ReverseProxy`? Assuming it's far off, is there any chance of getting a preview package published?

Specifically I'm looking to consume the recently added support for multiple configuration providers (#1534/#979) and would much prefer not maintaining a copy of the package myself.

**Comment by MihaZupan** [comment, marked as answer]
I'll let @samsp-msft comment on the timeline, but until then, you can consume [daily builds](https://github.com/microsoft/reverse-proxy/blob/main/docs/DailyBuilds.md).

**Reply by pinkfloydx33** [reply]
Doh. I looked all over for daily builds. Apparently I didn't look hard enough. That'll do it for now. Thanks!

**Reply by samsp-msft** [reply]
The goal is to ship 1.1 in Q1 this year, but we are functionality/quality driven rather than targeting a specific date.
